Dockerfile

```diff
@@ -16,10 +16,21 @@ RUN cd ./backend && go build -o ./server ./cmd/
 
 RUN cd ./frontend && npm install && npm run build
 
-# Create a plugins folder if none exists so we copy whatever exists to the
-# image.
+# Backwards compatibility, move plugin folder to only copy matching plugins.
+RUN mv plugins plugins-old
+
 RUN mkdir -p ./plugins
 
+# Backwards compatibility, copy any plugins found inside "./plugins" into ".plugins".
+# They should match plugins/MyFolder/main.js, otherwise they are not copied.
+RUN find plugins-old -maxdepth 2 -type f  | grep -i main.js$ | xargs -i dirname {} | xargs -i cp -a {} ./plugins/
```

Comment:
This should never be the case AFAIU. We assume that plugins-old will contain plugins in the old form of `MyPlugin/dist/main.js`, which should be covered by the other script you run below.

Reply:
plugins-old could also theoretically contain the [documented MyPlugin/main.js form](https://kinvolk.io/docs/headlamp/0.2/development/plugins/building/#shipping--deploying-plugins).

I wonder if we should support .plugins/MyPlugin/main.js and .plugins/MyPlugin/dist/main.js ? The dist form is useful because that's where the plugins are output by default, and then maybe people just point their plugins-dir at where they build their plugins.

Reply:
I removed the dist folder handling from the Dockerfile.